beacon-chain/p2p/encoder/ssz.go

```diff
@@ -18,6 +18,8 @@ var _ NetworkEncoding = (*SszNetworkEncoder)(nil)
 // MaxGossipSize allowed for gossip messages.
 var MaxGossipSize = params.BeaconNetworkConfig().GossipMaxSize // 1 Mib
 
+var MaxChunkSize = params.BeaconNetworkConfig().MaxChunkSize // 1 Mib
```

Comment:
```suggestion
var MaxChunkSize = params.BeaconNetworkConfig().MaxChunkSize
```

is it still 1Mib post merge? Perhaps we should remove since those type of comments are hard to maintain

Reply:
This is while initialising the system, post merge we should change the defaults to `GossipMaxSizeMerge` and `MaxChunkSizeMerge` which are 10 Mib. For now these comments makes sense i think.

Reply:
True. `MaxChunkSize` will always be 1Mib no matter what